### PR TITLE
[tesseract-ocr/tesstrain#288] updating urls to access raw content on github.com

### PR DESCRIPTION
**Pull request by z-aliakseyeu**
https://github.com/tesseract-ocr/tesstrain/issues/288

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -314,7 +314,7 @@ $(OUTPUT_DIR).traineddata: $(LAST_CHECKPOINT)
 endif
 
 $(DATA_DIR)/radical-stroke.txt:
-	wget -O$@ 'https://github.com/tesseract-ocr/langdata_lstm/raw/main/radical-stroke.txt'
+	wget -O$@ 'https://raw.github.com/tesseract-ocr/langdata_lstm/main/radical-stroke.txt'
```
Comment (stweil) [inline Makefile:317]:
```suggestion
	wget -O$@ 'https://raw.githubusercontent.com/tesseract-ocr/langdata_lstm/main/radical-stroke.txt'
```

File: Makefile [hunk 2]
```diff
@@ -353,7 +353,7 @@ tesseract-$(TESSERACT_VERSION):
 tesseract-langs: $(TESSDATA)/eng.traineddata
 
 $(TESSDATA)/eng.traineddata:
-	cd $(TESSDATA) && wget https://github.com/tesseract-ocr/tessdata$(TESSDATA_REPO)/raw/main/$(notdir $@)
+	cd $(TESSDATA) && wget https://raw.github.com/tesseract-ocr/tessdata$(TESSDATA_REPO)/main/$(notdir $@)
```
Comment (stweil) [inline Makefile:356]:
```suggestion
	cd $(TESSDATA) && wget https://raw.githubusercontent.com/tesseract-ocr/tessdata$(TESSDATA_REPO)/main/$(notdir $@)
```

**Review by stweil — CHANGES_REQUESTED** [review]
The URLs https://github.com/tesseract-ocr/langdata_lstm/raw/main/radical-stroke.txt and https://raw.github.com/tesseract-ocr/langdata_lstm/main/radical-stroke.txt are redirected to https://raw.githubusercontent.com/tesseract-ocr/langdata_lstm/main/radical-stroke.txt, so let us use that URL without further redirects.

**Comment by z-aliakseyeu** [comment]
@stweil :+1: will update now.

**Pull request closed without merging**